src/index.tsx

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import * as dotenv from 'dotenv';
+
+const app = express();
+const PORT = process.env.PORT || 5000;
+
+app.get('/', (_, res) => {
+	res.status(200).send();
+});

```

Comment:
Getting a 'Cannot Get /' when I run this branch locally. Were you able to get a success while testing? I'm testing in browser.